lib/bolt/module_installer.rb

```diff
@@ -155,7 +155,7 @@ def install(specs, path, moduledir, force: false, resolve: true)
         # and write a Puppetfile.
         if force || !path.exist?
           @outputter.print_action_step("Resolving module dependencies, this may take a moment")
-          puppetfile = Resolver.new.resolve(specs)
+          puppetfile = Resolver.new.resolve(specs, config)
```

Comment:
This we want to catch when it errors and install as if `--no-resolve` was passed ([per comment](https://github.com/puppetlabs/bolt/pull/2325#issuecomment-737528220))

Reply:
I think the only reasonable way to catch an error here is if we can distinguish between resolution errors and errors caused by a restricted network (I think that's the only kind of error a user might run into by `module-install` not being passed along, anyways). Considering a user will receive a warning that `module-install` does not work for resolution yet, and it links them to fairly detailed documentation on getting around that, I lean more towards letting this fail and the user deciding how to get around it.